### PR TITLE
Some documentation accessibility fixes

### PR DESCRIPTION
**Pull request by timwright12**
## Description of Change

Headings were out of order on the documentation homepage and the alt text on some images was redundant, so I fix those instances. Linting also ran over the file showing more changes to formatting.

### CODE REVIEW

File: VAMobile/documentation/docs/Engineering/BackEnd/Architecture/FeatureFlagging.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ You will then need to add your production id.me email address to `config/setting
 ## Toggling Feature Flags
 
 Flipper URLS (note the 'api' in the url)
-* staging: https://staging-api.va.gov/flipper/features
-* production: https://api.va.gov/flipper/features
+
+* staging: staging-api.va.gov/flipper/features
+* production: api.va.gov/flipper/features
```
Comment (timwright12) [inline VAMobile/documentation/docs/Engineering/BackEnd/Architecture/FeatureFlagging.md:25]: removing `https://` so they are not represented as links in the doc and rather URLs the should be read out

Comment (timwright12) [reply VAMobile/documentation/docs/Engineering/BackEnd/Architecture/FeatureFlagging.md:25]: @brea11y would like your take on this issue

**Comment by brea11y** [comment]
Hey @timwright12 – I agree with you on removing the https:// . If it is intentional for the full link to be read out, we can disregard masking the link.

**Review by kpethtel — APPROVED** [review]
👍

**Review by TimRoe — APPROVED** [review]

**Review by alexandec — APPROVED** [review]

File: VAMobile/documentation/docs/Engineering/BackEnd/Architecture/Endpoint Creation Checklist.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Some Lighthouse APIs use an authorization flow called, client credentials grant
 Adding a new service will require the following steps:
 (For questions on the first two steps, contact Derek Brown.)
   1. Request sandbox access [here](https://developer.va.gov/onboarding/request-sandbox-access) or ask Lighthouse to expand the permissions of an existing client id we have with them to include the new API. Client ids can be shared by anyone with the same VASI number. All of Vets-api uses the same VASI number so if possible, try to use an existing one. If a new client id is required, follow these steps:
```
[the remark this answers is not in the document]
Comment (timwright12) [reply VAMobile/documentation/docs/Engineering/BackEnd/Architecture/Endpoint Creation Checklist.md:48]: yeah, good call there. I'll update that

**Pull request merged**